src/Symfony/Component/Process/CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 CHANGELOG
 =========
 
+6.2.0
+-----
+
+* added support for `Process::findExecutable()` to find executables independent of `open_basedir` configuration
+
 5.2.0
 -----
 
- * added `Process::setOptions()` to set `Process` specific options
- * added option `create_new_console` to allow a subprocess to continue
-   to run after the main script exited, both on Linux and on Windows
+* added `Process::setOptions()` to set `Process` specific options
+* added option `create_new_console` to allow a subprocess to continue to run
+  after the main script exited, both on Linux and on Windows
```

Comment:
this should be reverted

Reply:
Fixed (don't know what I actually changed there)